src/ansys/additive/core/additive.py

```diff
@@ -55,6 +55,15 @@
 class Additive:
     """Provides the client interface to one or more Additive services.
 
+    In a typical cloud environment, a single Additive service with load balancing and
+    auto-scaling is used. The ``Additive`` client connects to the service via a
+    single connection. However, for atypical environments or when running on localhost,
+    the ``Additive`` client can perform crude load balancing by connecting to multiple
+    servers and distributing simulations across them. The ``server_connections``,
+    ``nsims_per_server``, and ``nservers`` parameters can be used to control the
```

Comment:
```suggestion
    `, ``nservers``, and ``nsims_per_server` parameters can be used to control the
```
Perhaps reverse the order in which the parameters are listed to match their descriptions later in the sentence.